docs/release-notes/v1.1.rst

```diff
@@ -1,6 +1,178 @@
+loop.
+
+A first step in the separation of the internal data model and the GADF format is introduced
+in this release in the handling of the pointing. This is part of a larger that will be
+implemented in the coming feature releases to allow I/O with multiple data formats.
+
+New features
+~~~~~~~~~~~~
+
+*gammapy.data*
+
+- A new function `~gammapy.data.get_irfs_features()` can extract the features of IRFs (such as energy
+  resolution bias or PSF radius) of a list of `Observation`. The output list can then be passed
+  to the function `~gammapy.utils.cluster.hierarchical_clustering()` which will find clusters
+  of IRF features allowing to combine `Observation` with similar IRF characteristics during
+  analysis.
+
+*gammapy.maps*
+
+- A `Map.reorder_axes` helper method has been introduced.
+- Dot product is now supported with the `Map.dot()` function. It applies the dot product on
+  the axes sharing the same name. It can also be called via the `@` operator.
+- A `WcsNDMap.to_region_nd_map_histogram()` helper method is introduced to compute
+  the histogram of the map values along the spatial dimensions for each non spatial axes
+  bins.
+
+*gammapy.modeling*
+
+- A new method has been added on `SpectralModel.spectral_index_error()` to compute
+  the spectral index at a given energy as well as its error by error propagation of
+  the covariance matrix elements.
+- The Franceschini (2018) and Saldana-Lopez (2021) EBL are now part of the built-ins
+  EBL models.
+- A spatial correction model can now added to `FoVBackgroundModel`.
+
+*gammapy.estimators*
+
+- Multiprocessing is now supported for `FluxPointsEstimator` and `LightCurveEstimator`. Setting
+  the number of cores used is done with the `n_jobs` property that is available on these
+  classes.
+
+*gammapy.visualization*
+
+- A function to plot `Map` as RGB is now proposed: `~gammapy.visualization.plot_map_rgb()`.
+- A function to plot the spectral distributions of predicted counts of models defined
+  on a `Dataset` is now available : `~gammapy.visualization.plot_npred_signal()`
+
+*documentation*
+
+- A new tutorial demonstrate how to use Gammapy with HAWC data.
+- The pulsar analysis tutorial now shows how to perform 3D analyses in phase.
+- A new tutorial demonstrating event sampling with energy dependent temporal models has
+  been added.
+
+*infrastructure*
+
+- Deprecation warnings can now be raised by Gammapy deprecated code. The warnings will appear
+  in the documentation and docstrings as well. Deprecated features will usually be removed
+  in the following feature release.
+- Minimal python version is now 3.9. Python 3.10 is supported as well.
+
+API changes
+~~~~~~~~~~~
+
+*gammapy.data*
+
+- the `GTI` now consists in a simple `Table` with `START` and `STOP` as `~astropy.time.Time`
+  object. `GTI.table` no longer contains a GADF formatted table with columns representing
+  start and stop time as METs (Mission Elpased Time). All methods should behave equivalently
```

Comment:
Is this an API change ?

Reply:
Well yes in the sense that `GTI.table` does not contain the same elements.